test/e2e/set-manifest-flags.ts

```diff
@@ -7,6 +8,25 @@ function parseIntOrUndefined(value: string | undefined): number | undefined {
   return value ? parseInt(value, 10) : undefined;
 }
 
+// Grab the tracesSampleRate from the git message if it's set
+function getTracesSampleRateFromGitMessage(): number | undefined {
```

Comment:
If the goal is to support this for specific PRs, is it possible to read this from the PR title or description via CircleCI?

Reply:
@matthewwalsh0 This would require network calls to the GitHub API, which if we don't want to do it hundreds of times, is best done in `get-changed-files-with-git-diff`

Let's see if anyone even wants to use this system before we spend a lot of additional time on it.